docs/articles/documentation/using-testcafe/configuration-file.md

```diff
@@ -203,53 +209,167 @@ You can use multiple reporters, but note that only one reporter can write to `st
 *API*: [runner.reporter](programming-interface/runner.md#reporter)
 
-## screenshotPath
+## screenshots
+
+Allows you to specify the screenshot options.
+
+### screenshots.path
 
-Enables screenshots and specifies the base directory where they are saved.
+Specifies the base directory where screenshots are saved.
 
 ```json
 {
-    "screenshotPath": "/home/user/tests/screenshots/"
+    "screenshots": {
+        "path": "/home/user/tests/screenshots/"
+    }
 }
 ```
 
 See [Screenshots](common-concepts/screenshots-and-videos.md#screenshots) for details.
 
-*CLI*: [-s, --screenshots](command-line-interface.md#-s-path---screenshots-path)  
+*CLI*: [--screenshots path](command-line-interface.md#path)  
 *API*: [runner.screenshots](programming-interface/runner.md#screenshots)
 
-## takeScreenshotsOnFails
+### screenshots.takeOnFails
 
 Specifies that a screenshot should be taken whenever a test fails.
 
 ```json
 {
-    "takeScreenshotsOnFails": true
+    "screenshots": {
+        "takeOnFails": true
+    }
 }
 ```
 
-Screenshots are saved to the directory specified in the [screenshotPath](#screenshotpath) option.
+Screenshots are saved to the directory specified in the [screenshots.path](#screenshotspath) property.
 
-*CLI*: [-S, --screenshots-on-fails](command-line-interface.md#-s---screenshots-on-fails)  
+*CLI*: [--screenshots takeOnFails](command-line-interface.md#takeonfails)  
 *API*: [runner.screenshots](programming-interface/runner.md#screenshots)
 
-## screenshotPathPattern
+### screenshots.pathPattern
 
 Specifies a custom pattern to compose screenshot files' relative path and name.
 
 ```json
 {
-    "screenshotPathPattern": "${DATE}_${TIME}/test-${TEST_INDEX}/${USERAGENT}/${FILE_INDEX}.png"
+    "screenshots": {
+        "pathPattern": "${DATE}_${TIME}/test-${TEST_INDEX}/${USERAGENT}/${FILE_INDEX}.png"
+    }
 }
 ```
 
 See [Path Pattern Placeholders](common-concepts/screenshots-and-videos.md#path-pattern-placeholders) for information about the available placeholders.
 
-> Use the [screenshotPath](#screenshotpath) option to enable screenshots.
+*CLI*: [--screenshots pathPattern](command-line-interface.md#pathpattern)  
+*API*: [runner.screenshots](programming-interface/runner.md#screenshots)
+
+### screenshots.fullPage
+
+Specifies that the full page should be captured, including content that is not visible due to overflow.
 
-*CLI*: [-p, --screenshot-path-pattern](command-line-interface.md#-p-pattern---screenshot-path-pattern-pattern)  
+```json
+{
+    "screenshots": {
+        "fullPage": true
+    }
+}
+```
+
+*CLI*: [--screenshots fullPage](command-line-interface.md#fullpage)  
 *API*: [runner.screenshots](programming-interface/runner.md#screenshots)
 
+## disableScreenshots
+
+Prevents TestCafe from taking screenshots.
+
+```json
+{
+    "disableScreenshots": true
+}
+```
+
+When this property is specified, screenshots are not taken whenever a test fails or a [screenshot action](../test-api/actions/take-screenshot.md) is executed.
```

Comment:
When this property is specified, screenshots are not taken when a test fails or a [screenshot action](../test-api/actions/take-screenshot.md) is executed.